src/Standards/Generic/Tests/CodeAnalysis/EmptyPHPStatementUnitTest.inc

```diff
@@ -52,3 +52,14 @@ function_call();
 <?php
 // Guard against false positives for two consecutive semi-colons in a for statement.
 for ( $i = 0; ; $i++ ) {}
+
+// Test for useless semi-colons
+for ( $i = 0; ; $i++ ) {};
+
+if (true) {};
+
+while (true) {};
+
```

Comment:
What do you mean ? It will be fix to
```
class ABC {}

switch ( $a ) {
    case 1:
        break;
    case 2:
        break;
    default:
        break;
}
```

Reply:
Not especially, but they weren't covered by your original PR ;-)

Reply:
Ok, more tests is always better. I added them.